packages/ses/src/module-link.js

```diff
@@ -13,14 +13,21 @@ import {
   makeModuleInstance,
   makeThirdPartyModuleInstance,
 } from './module-instance.js';
-import { entries, isArray } from './commons.js';
+import {
+  Error,
+  Map,
+  ReferenceError,
+  entries,
+  isArray,
+  isObject,
+  mapGet,
+  mapHas,
+  mapSet,
+  weakmapGet,
+} from './commons.js';
 
 const { quote: q } = assert;
 
-function isObject(o) {
-  return Object(o) === o;
-}
```

Comment:
Incidental refactor to reuse a shared implementation. In this case, the shared version is identical.